Sources/SwifterMedia.swift

```diff
@@ -60,22 +71,56 @@ public extension Swifter {
         let parameters = ["command": "FINALIZE",
                           "media_id": mediaId]
         self.postJSON(path: path, baseURL: .upload, parameters: parameters, success: { (json, response) in
+            guard let obj = json.object, obj.keys.count > 0 else {
+                let error = SwifterError(message: "Bad Response for Multipart Media Upload",
+                                         kind: .invalidMultipartMediaResponse)
+                failure?(error)
+                return
+            }
+            if let processingInfo = json["processing_info"].object, let state = processingInfo["state"]?.string {
+                // async FINALIZE response which requires further STATUS command call(s)
+                switch state {
+                case "pending", "in_progress":
+                    let secs = processingInfo["check_after_secs"]?.double ?? 3.0
```

Comment:
3 seconds seem like a really long time to retry. Can we not guarantee from the API that `check_after_secs` will always exist?

Reply:
Perhaps we can think that check_after_secs always returns at the time of pending and in_progress.